modules/Bio/EnsEMBL/Compara/RunnableDB/PairAligner/PairAligner.pm

```diff
@@ -145,53 +130,11 @@ sub fetch_input {
 		"  your system can cope with so many alignments.");
       }
   }
-  $self->compara_dba->dbc->disconnect_if_idle();
```

Comment:
Why is this not needed anymore?

Reply:
Because it was the last line in `fetch_input` and eHive will immediately reopen the connection to change the job status to RUN